puppet/manifests/install-java.pp

```diff
@@ -3,25 +3,30 @@
 node 'dev-box' {
 
 	# Will install the default JDK. On 20.04 this will install java-11-openjdk.
-	package { 'default-jdk':
-		ensure => installed,
+	#package { 'default-jdk':
+	#	ensure => installed,
+	#}
+
+	exec { 'java17':
+		command => '/usr/bin/apt-get install -y openjdk-17-jdk openjdk-17-jre',
```

Comment:
Optional: use https://sdkman.io/ to get access to exact/latest JDK's instead of os-supplied?